### DISCUSSION TITLE
questDB as a service

### DISCUSSION THREAD

**Discussion by ghost** [questdb/questdb#1688] [Ideas]
I would like to achieve something like yugabyte Cloud service, but using questDB.

QuestDb have plans for this or in the meantime any ideas on how to achieve a simillar experience?

The ideas is to autoscale quesdb by demand and pay as you go and don't need to have a headcount managaing this infrastruture

**Comment by bluestreak01** [comment]
Yes, we are working on a cloud product. Pay-as-you-go a little tricky for us as viable product. We're looking to provide more familiar infrastructure, e.g. choice of servers etc. However we're making it easy to scale these servers up and down with no downtime.
